geoapps/inversion/electricals/induced_polarization/pseudo_three_dimensions/constants.py

```diff
@@ -161,7 +158,7 @@
     },
     "chargeability_channel_bool": True,
     "chargeability_channel": {
-        "association": ["Cell", "Vertex"],
```

Comment:
Testing this on the FlinFlon_dcip.geoh5 - I couldn't pick the Line ID object saved in the DC Survey unless I restored the "vertex"

Reply:
Means we need to update the example.